### PR TITLE
feat(pagination): add shape (VIV-1062)

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

File: libs/components/src/lib/pagination/pagination.scss [hunk 1]
```diff
@@ -32,3 +31,12 @@
...
 }
+
+/* Shape */
+
+.control:not(.shape-pill) {
+	border-radius: 4px;
+}
+.control.shape-pill {
+	border-radius: 14px;
+}
```
Comment (rachelbt) [inline libs/components/src/lib/pagination/pagination.scss:42]: you can move this to be inside the control and use:

&:not(.shape-pill) {
	border-radius: 4px;
}
&.shape-pill {
	border-radius: 14px;
}

File: libs/components/src/lib/pagination/pagination.spec.ts [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
-import {Size} from '@vonage/vivid';
-import type {Button} from '../button/button';
-import {Pagination, PaginationSize} from './pagination';
+import { elementUpdated, fixture, getControlElement } from '@vivid-nx/shared';
+import { FoundationElementRegistry } from '@microsoft/fast-foundation';
+import type { Button } from '../button/button';
+import { Shape, Size } from '../enums';
+import { Pagination, PaginationShape, PaginationSize } from './pagination';
 import '.';
+import { paginationDefinition } from './definition';
```
Comment (rachelbt) [inline libs/components/src/lib/pagination/pagination.spec.ts:7]: I think we need to add the shape export in the definition as well

**Review by rachelbt — COMMENTED** [review]
LGTM
@AyalaBu if pill is set - are the next + previous button should also be pill?

File: libs/components/src/lib/pagination/pagination.spec.ts (hunk 2 above)
Comment (rinaok) [reply libs/components/src/lib/pagination/pagination.spec.ts:6]: @rachelbt added export to definition

**Review by rachelbt — APPROVED** [review]
LGTM

File: libs/components/src/lib/pagination/pagination.spec.ts (hunk 2 above)
Comment (AyalaBu) [reply libs/components/src/lib/pagination/pagination.spec.ts:6]: I believe so.
Putting together two different shapes in the pagination component
can create some real ugliness

**Pull request merged**